sycl/test-e2e/ESIMD/performance_tests/BitonicSortK.cpp

```diff
@@ -0,0 +1,15 @@
+//==---------------- BitonicSortK.cpp  - DPC++ ESIMD on-device test --------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu-intel-dg2
+
+// RUN: mkdir -p %t.dir && %{build} -o %t.dir/exec.out
+// RUN: env IGC_DumpToCustomDir=%t.dir IGC_ShaderDumpEnable=1 %{run} %t.dir/exec.out
+// RUN: python3 %S/instruction_count.py %t.dir 3452 VC_asmfc04983569d0d4c9__ZTSZZN11BitonicSort5SolveEPjS0_jENKUlRN4sycl3_V17handlerEE0_clES4_E5Merge.asm
+// RUN: rm -rf %t.dir
```

Comment:
We shouldn't need the `rm`, normal tests don't delete their output so we shouldn't need to do it here. The size of the shader dump is not large either.